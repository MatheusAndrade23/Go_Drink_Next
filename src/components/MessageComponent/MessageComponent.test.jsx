import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { MessageComponent } from '.';

describe('<MessageComponent />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <MessageComponent message="Wrong Password" />,
    );
    expect(container).toMatchSnapshot();
  });
});
