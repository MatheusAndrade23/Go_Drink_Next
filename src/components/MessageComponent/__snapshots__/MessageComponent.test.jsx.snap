// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageComponent /> should render 1`] = `
.c0 {
  position: fixed;
  background-color: #222831;
  padding: 40px;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 350px;
  text-align: center;
  z-index: 300;
  border: 2px solid #EEEEEE;
  border-radius: 10px;
  top: 70px;
  right: 70px;
}

.c0 p {
  color: #00ADB5;
  font-weight: bold;
  font-family: 'Montserrat',sans-serif;
}

@media (max-width:560px) {
  .c0 {
    right: 20px;
    left: 20px;
    width: auto;
  }
}

<div>
  <div
    class="c0"
  >
    <p>
      Wrong Password
    </p>
  </div>
</div>
`;
